### ISSUE TITLE
Add torque support

### ISSUE THREAD

**Issue by grzanka** [DataMedSci/mcpartools#7]
It looks that TORQUE (https://en.wikipedia.org/wiki/TORQUE) has similar commands to PBS (https://en.wikipedia.org/wiki/Portable_Batch_System).

**Comment by grzanka** [comment]
@ant6 access PBS via Zeus: https://docs.cyfronet.pl/pages/viewpage.action?pageId=4260614

**Comment by ant6** [comment]
```
[plg---@zeus mcpartools-0.1.4.post0.dev13]$ python -d mcpartools/generatemc.py -j 100 -p 1000 fluka.inp 
--version: srun: command not found
Traceback (most recent call last):
  File "mcpartools/generatemc.py", line 77, in <module>
    sys.exit(main(sys.argv[1:]))
  File "mcpartools/generatemc.py", line 71, in main
    ret_code = generator.run()
  File "/mnt/auto/people/plg---/mcpartools-0.1.4.post0.dev13/mcpartools/generator.py", line 78, in run
    self.generate_workspace()
  File "/mnt/auto/people/plg---/mcpartools-0.1.4.post0.dev13/mcpartools/generator.py", line 114, in generate_workspace
    jobdir_name = "job_{:04d}".format(jobid + 1)
ValueError: zero length field name in format
```

Probably that's because of old Python 2.6.6 on Zeus. Formatting was a bit different. https://docs.python.org/2.6/library/string.html#format-string-syntax - field name is required.

I changed it to:
`jobdir_name = "job_{0:04d}".format(jobid + 1)`  - `0` before `:04d`.
and now another one fails:
```
mcpartools-0.1.4.post0.dev13/mcpartools/mcengine/fluka.py", line 50, in randomize
    new_line = "RANDOMIZ         1.0 {:f}\n".format(new_seed)
ValueError: zero length field name in format
```
@grzanka I know this sound silly, but probably we have to support Python 2.6.

**Comment by ant6** [comment]
BTW - what is this? `{workspace_dir:s}/job_`printf %04d $SLURM_ARRAY_TASK_ID`/run.sh` in https://github.com/DataMedSci/mcpartools/blob/master/mcpartools/scheduler/data/run_slurm.sh#L7

What should I do for Torque?

**Comment by grzanka** [comment]
Wait - login to zeus and type `module spider python`

**Comment by grzanka** [comment]
Check also contents of `$PLG_GROUPS_SHARED/plggccbmc/software/setup.sh` and try to understand what is going on there.

**Comment by ant6** [comment]
Ok, 
```
  plgrid/tools/python:
----------------------------------------------------------------------------
     Versions:
        plgrid/tools/python/2.5.6
        plgrid/tools/python/2.6.9
        plgrid/tools/python/2.7.2-el5
        plgrid/tools/python/2.7.2-ucs4
        plgrid/tools/python/2.7.5
        plgrid/tools/python/2.7.9
        plgrid/tools/python/3.3.0
        plgrid/tools/python/3.3.2
        plgrid/tools/python/3.4.0
        plgrid/tools/python/3.4.2
```
That's better :P But still I would name formetted-strings - no cost, a bit more readability.

**Comment by ant6** [comment]
Ok, on Prometheus `module load tools/python/2.7.9` does the job.

**Comment by grzanka** [comment]
Name formatted strings OK, but I'd discuss if supporting python 2.6 is a good option.
Main cluster at CERN (called lxplus) is now switching from SLC6.8 (based on RedHat EL with python 2.6.6) to CentOS 7 (with python 2.7.5).

I'd recommend not to support python2.6, but for users not having access to new python make a release with nuitka (http://nuitka.net/) linked to 32bit python library. Such release (our binary + python libraries) will probably be heavy (50-100MB) but for cluster users space is usually not a problem.

**Comment by ant6** [comment]
No, python 2.6 is not a good idea indeed. I'll make separate issue for nuitka.

**Comment by ant6** [comment]
#49 done

**Comment by ant6** [comment]
Closed in #48
